src/current.rs

```diff
@@ -5,9 +5,15 @@
 //! example, a tool that wants to check linearizability might want access to a global timestamp for
 //! events, which the [`context_switches`] function provides.
```

Comment:
Could we add a paragraph to this doc comment with a quick overview of labels and what they're useful for?

Reply:
Added.